hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractOzoneFileSystemTest.java

```diff
@@ -174,7 +177,8 @@ abstract class AbstractOzoneFileSystemTest {
 
   @BeforeAll
   void init() throws Exception {
-    OzoneConfiguration conf = new OzoneConfiguration();
+    conf = new OzoneConfiguration();
+    conf.set("fs.o3fs.impl.disable.cache", "true");
```

Comment:
Please disable the cache only in the specific test case that needs new `FileSystem` with changed config, similar to:

https://github.com/apache/ozone/blob/264cbc61925761be9af96c9e22edefc1840af5ce/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/OzoneFileSystemTests.java#L63-L65